code/modules/projectiles/projectile/beams.dm

```diff
@@ -130,7 +130,9 @@
 
 /obj/item/projectile/beam/instakill/on_hit(atom/target)
 	. = ..()
-	if(iscarbon(target))
-		var/mob/living/carbon/M = target
-		M.visible_message("<span class='danger'>[M] explodes into a shower of gibs!</span>")
-		M.gib()
+	if(isliving(target))
+		var/mob/living/L = target
+		L.visible_message("<span class='danger'>[L] explodes!</span>")
+		L.gib()
+	else if(isturf(target) || istype(target,/obj/structure/))
+		target.ex_act(2)
```

Comment:
`ex_act(1)` is the most powerful explosion. Or did you want something more moderate?